awswrangler/s3/_write_parquet.py

```diff
@@ -510,6 +540,13 @@ def to_parquet(  # pylint: disable=too-many-arguments,too-many-locals
         catalog_table_input = catalog._get_table_input(  # pylint: disable=protected-access
             database=database, table=table, boto3_session=session, catalog_id=catalog_id
         )
+        if path is None:
+            if catalog_table_input is not None:
+                path = catalog_table_input["StorageDescriptor"]["Location"]
+            else:
+                raise exceptions.InvalidArgumentValue(
+                    "Glue table does not exist. Please pass the `path` argument to create it."
+                )
```

Comment:
Path is now an optional argument if the Glue table already exists. It's required otherwise